src/coreclr/jit/gentree.cpp

```diff
@@ -17807,6 +17807,20 @@ bool GenTree::isContainableHWIntrinsic() const
             return true;
         }
 
+        default:
+        {
+            return false;
+        }
+    }
+#elif TARGET_ARM64
+    switch (AsHWIntrinsic()->GetHWIntrinsicId())
+    {
+        case NI_Vector64_get_Zero:
+        case NI_Vector128_get_Zero:
```

Comment:
Not important to handle in this PR, but we should probably have `morph` or some other phase recognize other types of `Zero` and normalize them to `get_Zero`.

This includes cases like `Vector64.Create(0)` and `Vector64.Create(0, 0, 0, ...)`

Reply:
Made an issue: https://github.com/dotnet/runtime/issues/63490
I could take a look at it once we iron out most of the uses cases of `Vector64/Vector128.Zero`.